website/docs/general/security-guide.md

```diff
@@ -1,6 +1,19 @@
 ---
 id: security
 title: Security
+keywords:
+- APISIX
+- api6
+- API gateway
+- API hosting
+- Security
+- Apache APISIX
+- apisix gateway
+- Ingress Controller
+- Service Mesh
+- Microservices
+- Release Guide
```

Comment:
found nothing about release guide

Reply:
removed

Reply:
Do we still need this?

Reply:
Yes, I suggest keep these keywords for now. Adjust them based on performance later.